packages/grid/_modules_/grid/constants/i18nConstants.ts

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT_LOCALE_TEXT = {
```

Comment:
How about we store that in a separate JSON file? So we can a file per culture?

Reply:
Sure, where would you want to store that json? We can keep it in the `constants/` folder for now?

Reply:
@dtassone You should find more details on the linked issue #196. The globalization topic was split into two phases 1. internationalization (to empower localization) 2. localization. For now, we are working on the internationalization infrastructure (1.).

Your comment is on 2. the plan was to do it later. We wanted to open a new issue for it once #196 is closed with this pull request. We can continue the conversation. For 2, we should be able to empower the same approach to what we do with the other components for consistency.

Reply:
> Sure, where would you want to store that json? We can keep it in the constants/ folder for now?

Please don't, not yet.

Reply:
I'm resolving - the outcome is that we will keep it in a js file because we may need to use either functions or nodes for some of the cases